### ISSUE TITLE
farmr error in the logs. Been finding it on all the chains.

### ISSUE THREAD

**Issue by Yamanipanuchi** [guydavis/machinaris#476]
```
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
```

**Comment by panayotovip** [comment]
same here and fullnode go not synced (machinaris 0.6.7)

**Comment by panayotovip** [comment]
After restarting the container blockchain start to sync.
2 hours later again stop syncing and get in logs :
```
reply: 'HTTP/1.1 200 OK\r\n'
header: Server: gunicorn
header: Connection: close
header: Content-Type: text/html; charset=utf-8
header: Content-Length: 17
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
error: stat of /root/.chia/farmr/log*txt failed: No such file or directory
```
now list in that directory (/root/.chia/farmr/) is :
```
4.0K drwxr-sr-x 2 root users 4.0K Dec  2 20:22 blockchain
4.0K drwxr-sr-x 2 root users 4.0K Dec  5 12:20 cache
4.0K drwxr-sr-x 2 root users 4.0K Dec  2 20:22 config
4.0K -rw-r--r-- 1 root users   48 Dec  2 20:22 id.json
 48K -rw-r--r-- 1 root users  47K Dec  5 12:20 log-xch.txt.1
 68K -rw-r--r-- 1 root users  68K Dec  2 20:26 log-xch.txt.2
4.0K -rw-r--r-- 1 root users   60 Dec  5 12:15 log.txt.1
4.0K -rw-r--r-- 1 root users   60 Dec  2 20:25 log.txt.2
4.0K -rw-r--r-- 1 root users   31 Dec  5 11:04 override-xch-binary.txt
```
There is missing log*txt files. 
I really hope that help to resolve the issue.

**Comment by guydavis** [comment]
Thanks.  Please try [upgrading all your containers](https://github.com/guydavis/machinaris/wiki/Releases#release-streams) to `:test` as this spurious log message has already been addressed there.

**Comment by ahowlett1965** [comment]
I have the same message, but I also get CPU use under 2%. It was up to 80%, but no longer seems to be plotting.

**Comment by guydavis** [comment]
Marking as fixed in [v0.6.8](https://github.com/guydavis/machinaris/releases/tag/0.6.8).  Based on user feedback after using Farmr for a month, I have set `farmr` as **DISABLED** by default.  One must specifically enable `farmr` if one really wants such [additional monitoring](https://github.com/guydavis/machinaris/wiki/Monitoring#farmr). Thanks again!
